mantis-control-plane/mantis-control-plane-client/src/main/java/io/mantisrx/server/master/client/TestGetMasterMonitor.java

```diff
@@ -56,8 +56,7 @@ public static void main(String[] args) {
         final CountDownLatch latch = new CountDownLatch(5);
         StaticPropertiesConfigurationFactory configurationFactory = new StaticPropertiesConfigurationFactory(properties);
         CoreConfiguration config = configurationFactory.getConfig();
-        final CuratorService curatorService = new CuratorService(config);
```

Comment:
This is a change that might be controversial. The CuratorService and really the CuratorFramework from IMO is really Zookeeper. In order to make the abstraction around Zookeeper I chose to reduce the exposure and knowledge of the curator to inside the ZK leadership functions.